daemon/docker.go

```diff
@@ -83,11 +83,19 @@ func deploy(repo *git.Repository, cli *docker.Client, out io.Writer) error {
 	// See https://cloud.google.com/community/tutorials/docker-compose-on-container-optimized-os
 	fmt.Fprintln(out, "Building project...")
 	ctx := context.Background()
+
+	repoName, err := common.GetProjectName(repo)
+	if err != nil {
+		fmt.Println(repoName)
+		return err
+	}
+
 	resp, err := cli.ContainerCreate(
 		ctx, &container.Config{
 			Image:      dockerCompose,
 			WorkingDir: "/build/project",
 			Cmd: []string{
+				"-p", repoName,
```

Comment:
Try updating this whole section:
```go
	resp, err := cli.ContainerCreate(
		ctx, &container.Config{
			Image:      dockerCompose,
			WorkingDir: "/build/project",
			Env:        []string{"HOME=/build"},
			Cmd: []string{
				"up",
				"--build",
			},
		},
		&container.HostConfig{
			Binds: []string{
				"/var/run/docker.sock:/var/run/docker.sock",
				os.Getenv("HOME") + ":/build",
			},
		}, nil, "docker-compose",
	)
```

Unfortunately I think I broke it a little while back 😓